[DOC-976][ybm] New track names in Aeon (#31384)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-cloud/cloud-basics/create-clusters-overview.md

### YugabyteDB version

Clusters are created using a stable release, taken from the [stable release series](../../../releases/versioning/#stable-releases) of YugabyteDB. You can choose to deploy your dedicated cluster using the following tracks:

- Extended - Has less frequent updates, using mature [LTS releases](../../../releases/versioning/#stable-release-support-policy) that have been tested longer in YugabyteDB Aeon.
- Rapid - Updated more frequently, providing quicker access to new features. Rapid can include [STS and LTS releases](../../../releases/versioning/#stable-release-support-policy), depending on which is more recent.

{{< note title="Production, Innovation, and Early Access tracks" >}}

> Clusters previously on the Innovation and Production tracks will move to the Extended track.

Reviewer: ```suggestion
Clusters previously on the Innovation and Production tracks are now on the Extended track.
```

Clusters previously on the Innovation and Early Access tracks will move to the Rapid track.

> 

Reviewer 2: Clusters on Innovation and Production will be moved to Extended track.
Clusters on preview and Early Access track will be moved to Rapid track.
So please change the note accordingly.

> Clusters previously on the Preview and Early Access tracks will move to the Rapid track.

Reviewer: ```suggestion
Clusters previously on the Preview and Early Access tracks are now on the Rapid track.
```
